marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/jobs.sjs

```diff
@@ -183,6 +184,14 @@ class Jobs {
     if (batchStatus === "finished" || batchStatus === "finished_with_errors" || batchStatus === "failed") {
       docObj.batch.timeEnded = fn.currentDateTime();
     }
+    for (const [key, value] of Object.entries(stackTraceObj)) {
```

Comment:
I believe this can be simplified with  "docObj.batch.fileName = stackTraceObj.uri" and "docObj.batch.lineNumber = stackTraceObj.line"